docs/pages/database-access/guides/mysql-aws.mdx

```diff
@@ -0,0 +1,278 @@
+
+## Create Database User
+
+Database accounts must have IAM authentication enabled in order to be allowed
+access. For MySQL:
+
+```sql
+CREATE USER alice IDENTIFIED WITH AWSAuthenticationPlugin AS 'RDS';
+```
+
+See [Creating a database account using IAM authentication](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/UsingWithRDS.IAMDBAuth.DBAccounts.html)
+for more information.
+
+## Configure Teleport
+
+Teleport Database Access is available starting from `6.0.0-alpha.1` pre-release.
+
+Download the appropriate version of Teleport for your platform from the table
+below, or visit our [downloads page](https://goteleport.com/teleport/download).
+
+{!docs/pages/preview/releases-table.mdx!}
+
+<Admonition type="warning" title="Version Warning">
+Note, pre-releases are not suitable for production usage!
+</Admonition>
+
+Follow the installation [instructions](../../installation.mdx).
+
+### Start Auth/Proxy Service
+
+Create a configuration file for a Teleport service that will be running
+auth and proxy servers:
+
+```yaml
+teleport:
+  data_dir: /var/lib/teleport
+  nodename: test
+auth_service:
+  enabled: "yes"
+  cluster_name: "test"
+  listen_addr: 0.0.0.0:3025
+  tokens:
+  - proxy,node,db:cbdeeab9-6f88-436d-a673-44d14bd86bb7
+proxy_service:
+  enabled: "yes"
+  listen_addr: 0.0.0.0:3023
+  web_listen_addr: 0.0.0.0:3080
+  tunnel_listen_addr: 0.0.0.0:3024
+  public_addr: teleport.example.com:3080
+  # MySQL proxy is listening on a separate port and needs to be enabled
+  # on the proxy server.
+  mysql_listen_addr: 0.0.0.0:3036
+ssh_service:
+  enabled: "no"
+```
+
+Start the service:
+
+```sh
+$ teleport start --debug --config=/path/to/teleport.yaml
+```
+
+### Start Database Service with CLI Flags
+
+For a quick try-out, Teleport database service doesn't require a configuration
+file and can be launched using a single CLI command:
+
+```sh
+$ teleport start --debug \
+   --roles=db \
+   --token=cbdeeab9-6f88-436d-a673-44d14bd86bb7 \
+   --auth-server=teleport.example.com:3080 \
+   --db-name=aurora \
+   --db-protocol=mysql \
+   --db-uri=mysql-aurora-instance-1.xxx.us-east-1.rds.amazonaws.com:3306 \
+   --db-aws-region=us-east-1 \
+   --labels=env=dev
+```
+
+Note that the `--auth-server` flag must point to the Teleport cluster's proxy endpoint
+because the database service always connects back to the cluster over a reverse
+tunnel.
+
+Instead of using a static auth token, a short-lived dynamic token can also
+be generated for a database service:
+
+```sh
+$ tctl tokens add \
+    --type=db \
+    --db-name=test \
+    --db-protocol=mysql \
+    --db-uri=db.example.com:3306
+```
+
+### Start Database Service with Config File
+
+Below is an example of a database service configuration file that proxies
+a single AWS Aurora MySQL database:
+
+```yaml
+teleport:
+  data_dir: /var/lib/teleport-db
+  nodename: test
+  # Auth token to connect to the auth server.
+  auth_token: cbdeeab9-6f88-436d-a673-44d14bd86bb7
```

Comment:
let's never ever hardcode any tokens in our docs, you will find those everywhere in the internet

instead promote usage the most secure tokens - dynamic tokens with short ttl.

```
tctl nodes add --roles=db > /tmp/file
```

and then join
```
tctl start --token=/tmp/file
```